eng/Versions.props

```diff
@@ -31,7 +31,7 @@
     <MicrosoftExtensionsFileSystemGlobbingVersion>2.0.0</MicrosoftExtensionsFileSystemGlobbingVersion>
     <MicrosoftFakesVersion>17.4.0-beta.22478.3</MicrosoftFakesVersion>
     <MicrosoftInternalCodeCoverageVersion>17.8.5-beta.23422.2</MicrosoftInternalCodeCoverageVersion>
-    <MicrosoftNetCompilersToolsetVersion>4.7.0-3.23315.4</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>4.8.0-1.23419.1</MicrosoftNetCompilersToolsetVersion>
```

Comment:
Are you sure that you need to reference the toolset compiler package at all? Isn't the compiler from the SDK good enough for VSTest?

Reply:
We started to have some build time issue about some analyzer not working because of the version of compiler (I don't recall precisely) and this was the only workaround I found that worked. I will try to remove that and check if that fails.

Reply:
@ViktorHofer Error is CSC(0,0): error CS9057: (NETCORE_ENGINEERING_TELEMETRY=Build) The analyzer assembly 'D:\a\_work\1\s\.dotnet\sdk\8.0.100-rtm.23506.1\Sdks\Microsoft.NET.Sdk\codestyle\cs\Microsoft.CodeAnalysis.CodeStyle.dll' references version '4.8.0.0' of the compiler, which is newer than the currently running version '4.7.0.0'.

Reply:
I would just remove this property and the one in L20 as you shouldn't need a newer compiler than what the SDK offers.